Moonlight macOS/Utility/keyboardTranslation.m

```diff
@@ -0,0 +1,163 @@
+//
+//  keyboardTranslation.m
+//  Moonlight macOS
+//
+//  Created by Felix Kratz on 10.03.18.
+//  Copyright © 2018 Felix Kratz. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "keyboardTranslation.h"
+#import <Limelight.h>
+
+CGKeyCode modifierKeyFromEvent(int keyModifier) {
+    switch (keyModifier) {
+        case 131330:
+            return 0xA0; //LSHIFT
+            //case 131332:              TODO: This will lockup the modifiers
+        //    return 0xA1; //RSHIFT
+        case 524576:
+            return 0xA4; //LALT
+        //case 524608:                  TODO: This will lockup the modifiers
+        //    return 0xA5; //RALT
+        case 262401:
+            return 0xA2; //LCTRL
+
+        default:
+            return 0x00;
+    }
+}
+
+char keyModifierFromEvent(int keyModifier)
```

Comment:
`keyModifierFromEvent` and `modifierKeyFromEvent` have names are easy to confuse with each other. Can you rename one or both?

Reply:
I decided to rename both, maybe this could also be the foundation for a virtual keyboard on ios? I don't know if the keyCodes are the same though.